code/__defines/misc.dm

```diff
@@ -208,3 +208,5 @@
 #define EMAIL_BROADCAST "broadcast@internal-services.net"
 
 #define LEGACY_RECORD_STRUCTURE(X, Y) GLOBAL_LIST_EMPTY(##X);/datum/computer_file/data/##Y/var/list/fields[0];/datum/computer_file/data/##Y/New(){..();GLOB.##X.Add(src);}/datum/computer_file/data/##Y/Destroy(){. = ..();GLOB.##X.Remove(src);}
+
+#define WORLD_ICON_SIZE 32
```

Comment:
This can be accessed with `world.icon_size`, why make it a define?

Reply:
On /vg/ it's a define because of constant expressions. To be honest I flat forgot about `world.icon_size` when writing this so I'll replace this with that.